crates/key-server/src/server.rs

```diff
@@ -504,7 +511,10 @@ async fn handle_fetch_key_internal(
         .await.tap_ok(|_| info!(
             "Valid request: {}",
             json!({ "user": payload.certificate.user, "package_id": valid_ptb.pkg_id(), "req_id": req_id, "sdk_version": sdk_version })
-        ))
+        )).tap_err(|e| match e {
```

Comment:
wouldn't it be better in line 543 (handle_fetch_key)?

Reply:
Sure, that could work too. I just wanted the log message for valid and invalid requests to be done the same place.